corehq/apps/cloudcare/views.py

```diff
@@ -220,7 +221,26 @@ def form_context(request, domain, app_id, module_id, form_id):
 cloudcare_api = login_or_digest_ex(allow_cc_users=True)
 
 
+def get_cases_vary_on(request, domain):
+    return [
+        request.couch_user.get_id
+        if request.couch_user.is_commcare_user() else request.REQUEST.get('user_id', ''),
+        request.REQUEST.get('ids_only', 'false'),
+        request.REQUEST.get('case_id', ''),
+        request.REQUEST.get('footprint', 'false'),
+        request.REQUEST.get('include_children', 'false'),
+        request.REQUEST.get('closed', 'false'),
+        json.dumps(get_filters_from_request(request)),
+        domain,
+    ]
+
+
+def get_cases_skip_arg(request, domain):
+    return string_to_boolean(request.REQUEST.get('cache_bust', 'true'))
+
+
 @cloudcare_api
+@skippable_quickcache(get_cases_vary_on, get_cases_skip_arg, timeout=50 * 60)
 def get_cases(request, domain):
```

Comment:
actually maybe not, if i were to do that i should put the `skippable_quickcache` on the `get_filtered_cases` function. if i do that i don't think it'll work anymore because it'll cache the case list. it may be best to leave this where it is because it more clearly denotes the fact that all i want to do is cache the specific route